common/nexus/failure.go

```diff
@@ -219,8 +219,8 @@ func HandlerErrorTypeFromHTTPStatus(statusCode int) nexus.HandlerErrorType {
 		return nexus.HandlerErrorTypeNotImplemented
 	case http.StatusServiceUnavailable:
 		return nexus.HandlerErrorTypeUnavailable
-	case nexus.StatusDownstreamTimeout:
-		return nexus.HandlerErrorTypeDownstreamTimeout
+	case nexus.StatusUpstreamTimeout:
```

Comment:
`HandlerErrorTypeFromHTTPStatus` shouldn't be required anymore, this is now part of the Nexus SDK.

Reply:
handler.go:

```
handlerErr := &nexus.HandlerError{
		Type:    commonnexus.HandlerErrorTypeFromHTTPStatus(resp.StatusCode),
		Failure: &failure,
	}
```
	
What did you have in mind for this, if we remove the statusCode --> errorType lookup?

Reply:
The Nexus SDK will now do this translation and give you a `HandlerError` back instead of an `UnexpectedResponseError`. You don't need to construct the `HandlerError` yourself anymore.